README.md

```diff
@@ -73,6 +73,19 @@ The jaeger-mongodb plugin uses the [grpc] storage architecture to interface with
 
 6. You can then play around with the HotROD application, and the traces will be stored to the local MongoDB database that we previously set up. The traces will reflect in the Jaeger UI. If you have MongoDB compass install, you can connect to it to visualize the data.
 
+## Configurable Options
+- Below is a list of options where you can configure with the mongodb storage plugin.
+- To customize any options, set the `GRPC_STORAGE_PLUGIN_CONFIGURATION_FILE` environment variable to point to a config.yaml file with options (for example, refer to `/configs/example.config.yaml` ).
```

Comment:
`example.config.yaml` should be`example-config.yaml`

Reply:
resolved